webui/react/Makefile

```diff
@@ -88,4 +88,4 @@ push-shared: test-shared
 
 	# fail if there are uncommitted changes.
 	git diff --exit-code || (echo "uncommitted changes, please commit or stash" ; false)
-	cd $(git rev-parse --show-toplevel) && git subtree push --prefix=webui/react/src/shared https://github.com/determined-ai/shared-web main
+	cd $(shell git rev-parse --show-toplevel) && git subtree push --prefix=webui/react/src/shared https://github.com/determined-ai/shared-web main
```

Comment:
what's the different with and without `shell`?

Reply:
shell run the arguments as a shell command. without it the contents get evaluted to "" not sure exactly how make evalutes the args. anyway this is supposed to go in through another pr https://github.com/determined-ai/determined/pull/4672 I'll take it out from here